### PR TITLE
Minor typo in library-foldertoolbar-view.

### PR DESCRIPTION
**Pull request by dannon**
Minor typo (not a functional problem since the method name was consistent).

Included a full client-format in https://github.com/galaxyproject/galaxy/commit/e8dad5c361abf0c270a5628b39e635d9641fbe6c to re-sync.

### CODE REVIEW

File: client/galaxy/scripts/mvc/toolshed/repositories-view.js [hunk 1]
```diff
@@ -114,45 +123,45 @@ var ToolShedCategoryContentsView = Backbone.View.extend({
...
             '<a class="fa">&hellip;</a>',
-            '<% } %>',
-            '<% } else { %>',
+            "<% } %>",
+            "<% } else { %>",
             '<a data-page="1" class="pagenav-inactive fa"><a>1</a>',
-            '<% } %>',
-            '<% _.each(page_slice, function(i) { %>',
-            '<% if (i == page) { %>',
+            "<% } %>",
```
Comment (martenson) [inline client/galaxy/scripts/mvc/toolshed/repositories-view.js:135]: lol, why prettier, why?!

Comment (dannon) [reply client/galaxy/scripts/mvc/toolshed/repositories-view.js:135]: Consistency in quoting!  :)

Comment (martenson) [reply client/galaxy/scripts/mvc/toolshed/repositories-view.js:135]: nope, it leaves the above line with `'`, if anything this decreases consistency

Comment (dannon) [reply client/galaxy/scripts/mvc/toolshed/repositories-view.js:135]: Ahh, I see what you mean.  Yeah, if there's a conflict in the simple substitution of the outer quotes (so, here, the quotes in the string are `"` as well) it won't mangle the contents of your string obviously.

Comment (martenson) [reply client/galaxy/scripts/mvc/toolshed/repositories-view.js:135]: right...the consistency is everpresent in this approach 😭

Comment (dannon) [reply client/galaxy/scripts/mvc/toolshed/repositories-view.js:135]: Multiple layers of consistency!  At least when everyone runs prettier it comes out the same :)

**Pull request merged**